Motorcycle.cpp

```diff
@@ -17,11 +18,14 @@ void Motorcycle::setSpeed(int s)
 {
     if( s < 90 )
     {
+        Vehicle::setSpeed(110);
```

Comment:
if `s < 90` this function ignores adjusting the speed to below 90.  you don't need this call to the base class function.
- [ ] remove it.